### DISCUSSION TITLE
Meshcentral fullscreen on 4k monitor

### DISCUSSION THREAD

**Discussion by impedator** [Ylianst/MeshCentral#5868] [Issues]
Hello!

I'm using meshcentral for a few years without any issue, but now I've change monitor to panoramic 4k - 3440x1440 px and full screen mode is working only on left side of the monitor in firefox
![2024-02-27 21_52_57-Window](https://github.com/Ylianst/MeshCentral/assets/6593087/1847c9f2-2486-4a5c-b83b-714b14770d29)

do you know why? there is no such behavior in edge or chromium, only firefox.

**Comment by si458** [comment]
firefox cant handle 4k maybe? if it works in chrome/edge then i think its a firefox bug
